scripts/webpack/plugin-factory.js

```diff
@@ -47,6 +48,11 @@ class PluginFactory {
   createCssExtractorPlugin(outputFilenamePattern) {
     return new ExtractTextPlugin(outputFilenamePattern);
   }
+
+  getProjectVersion_() {
+    const lernaJson = require('../../lerna.json');
```

Comment:
Is this going to work as expected? Don't we run lerna publish which revs things _after_ running pre-release.sh which would run this code?

Reply:
Good catch, Ken. I updated the PR to fix that by refactoring the release scripts a bit.